src/Symfony/Bridge/Doctrine/Form/Type/EntityType.php

```diff
@@ -76,7 +76,7 @@ public function getBlockPrefix()
      * @internal This method is public to be usable as callback. It should not
      *           be used in user code.
      */
-    public function getQueryBuilderPartsForCachingHash($queryBuilder): ?array
+    public function getQueryBuilderPartsForCachingHash(object $queryBuilder): ?array
```

Comment:
```suggestion
    public function getQueryBuilderPartsForCachingHash(QueryBuilder $queryBuilder): ?array
```

The first if statement can also be removed

Reply:
actually not possible because of the parent class and LSP